### PR TITLE
Add FlattenTransparency Kotlin sample

### PR DESCRIPTION
**Pull request by datalogics-saharay**
Let's add a FlattenTransparency Kotlin sample to our collection of Kotlin samples!

### CODE REVIEW

File: FlattenTransparency/src/main/kotlin/com/datalogics/pdfl/samples/FlattenTransparency.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+/*
+ * This sample shows how to flatten transparencies in a PDF document.
+ *
+ * PDF files can have objects that are partially or fully transparent, and thus
+ * can blend in various ways with objects behind them. Transparent graphics or images
+ * can be stacked in a PDF file, with each one contributing to the final result that
+ * appears on the page. The process to flatten a set of transparencies merges them
+ * into a single image on the page.
+ *
+ * Copyright (c) 2007-2024, Datalogics, Inc. All rights reserved.
```
Comment (datalogics-robb) [inline FlattenTransparency/src/main/kotlin/com/datalogics/pdfl/samples/FlattenTransparency.kt]: Should this copyright say 2007-2024 or should it just say 2024?

Comment (datalogics-saharay) [reply FlattenTransparency/src/main/kotlin/com/datalogics/pdfl/samples/FlattenTransparency.kt]: @datalogics-robb The Java sample has 2007 - 2023 as the copyright, so I just modified that for 2024. I'm alright with either, but 2024 might be the way to go.

Comment (datalogics-robl) [reply FlattenTransparency/src/main/kotlin/com/datalogics/pdfl/samples/FlattenTransparency.kt]: The copyright is supposed to cover the time the file was created until the present day, so this should just be "2024" (and get bumped to "2024-2025" next year).

@datalogics-robb As a side issue, we need to bump the copyright dates in our other samples.

Comment (datalogics-saharay) [reply FlattenTransparency/src/main/kotlin/com/datalogics/pdfl/samples/FlattenTransparency.kt]: I updated the copyright date on the Kotlin samples.

**Review by datalogics-robl — APPROVED** [review]

**Pull request merged**